liblwgeom/lwgeom_geos.c

```diff
@@ -32,30 +31,26 @@
 #include <stdlib.h>
 #include <time.h>
 
-LWTIN *lwtin_from_geos(const GEOSGeometry *geom, int want3d);
+LWTIN* lwtin_from_geos(const GEOSGeometry* geom, int want3d);
 
 #undef LWGEOM_PROFILE_BUILDAREA
 
 #define LWGEOM_GEOS_ERRMSG_MAXSIZE 256
 char lwgeom_geos_errmsg[LWGEOM_GEOS_ERRMSG_MAXSIZE];
 
 extern void
-lwgeom_geos_error(const char *fmt, ...)
+lwgeom_geos_error(const char* fmt, ...)
 {
 	va_list ap;
-
 	va_start(ap, fmt);
 
 	/* Call the supplied function */
-	if ( LWGEOM_GEOS_ERRMSG_MAXSIZE-1 < vsnprintf(lwgeom_geos_errmsg, LWGEOM_GEOS_ERRMSG_MAXSIZE-1, fmt, ap) )
-	{
-		lwgeom_geos_errmsg[LWGEOM_GEOS_ERRMSG_MAXSIZE-1] = '\0';
-	}
+	if (LWGEOM_GEOS_ERRMSG_MAXSIZE - 1 < vsnprintf(lwgeom_geos_errmsg, LWGEOM_GEOS_ERRMSG_MAXSIZE - 1, fmt, ap))
```

Comment:
Why are you removing braces?

Reply:
Yeah, there is quite some opinionated re-styling here. The "good neighbour" style policy is to try to ape what's already there, little though you make like it, rather than bringing everything into line w/ your own personal preferences.

Reply:
@Komzpa can you comment?